docs/dsp-tools-usage.md

```diff
@@ -254,61 +254,45 @@ In order to upload data incrementally the procedure described [here](dsp-tools-x
 and shutdown of API and APP can be complicated: Both repos need to be cloned locally, a `git pull` has to be executed 
-from time to time to stay up to date, and then there are several commands for each repository to remember. 
-
-Another challenge is the software that DSP depends upon: JDK, node, npm, Angular, etc. should be kept up to date. And
-it might happen that a dependency is replaced, e.g. JDK 11 Zulu by JDK 17 Temurin. An non-developer can quickly get lost
-in this jungle. 
-
-That's why dsp-tools offers some commands to facilitate the handling of API and APP. These commands 
-
- - clone the repos to `~/.dsp-tools`, and keep them up to date.
- - check every time if the dependencies are up to date, and give you advice how to update them, if necessary.
- - pass on the right commands to APP and API, even if the correct usage of these commands changes over time.
- - make sure that the repos don't get cluttered with old files over time.
- - log their activity in `~/.dsp-tools`, so you can check the logs for troubleshooting, if necessary.
-
-The only requirements for these commands are:
-
- - the initial installation of all software that you accomplished when you started working at DaSCH
- - Docker must be running (for DSP-API only)
-
-Please note that these commands were developed for DaSCH-internal use only. They only work on Macs that have the 
-required software installed that makes it possible to run the API and APP. We don't offer support or troubleshooting 
-for these commands.
-
+from time to time to stay up to date, and then there are several commands for each repository to remember. And your 
+local clone of the API might get cluttered with old files over time.
 
-### Start DSP-API
+Another challenge is the software that DSP depends upon, and that should be kept up to date: JDK, node, npm, Angular, 
+etc. And it might happen that a dependency is replaced, e.g. JDK 11 Zulu by JDK 17 Temurin. A non-developer can quickly 
+get lost in this jungle. 
 
+That's why dsp-tools offers a command to facilitate the handling of API and APP:
 ```
-dsp-tools start-api
+dsp-tools start-stack
 ```
 
-This command makes a clone of the [DSP-API repository](https://github.com/dasch-swiss/dsp-api) into `~/.dsp-tools`. If
-it finds an existing clone there, it runs `git pull` instead. If the API is already running, it shuts down the old 
-instance, deletes all data that was in it, and starts a new one. If the dependencies are outdated or not installed, a 
-warning is printed to the console.
-
-
-### Shut DSP-API down
+This calls Docker commands to start up the latest deployed versions of API and the APP, i.e. the versions that are 
+running on [https://admin.dasch.swiss](https://admin.dasch.swiss). It's no longer necessary to have two local clones, 
+nor to execute `make` commands inside them. The only requirement for this command is that Docker must be running. It 
+can even be executed on a Windows machine without any of the software dependencies.
```

Comment:
maybe I just overlooked it... but you'd still have docker installed&running as a requirement - so that should definitely be explicitly mentioned.

Reply:
also, speaking of requirements: do we write anywhere that python (and which python version) must be installed? just came to my mind...

Reply:
dsp-tools is a pip package, so it's difficult to have it without Python. setup.py says `python_requires=">=3.9.0"`, so we already have it pinned there. But actually, I don't think that dsp-tools really requires a certain Python version, does it?

Reply:
yes... don't ask me why... but I remember Eric at some point having an issue which turned out to be him having 3.8 installed. So probably it would be good to specify that somewhere as I don't think pip will stop you from installing it on a lower python version (not sure though). But that's by no means urgent.